### PR TITLE
Fixes client lag on staging

### PR DESCRIPTION
**Pull request by MaxIsJoe**
__nocl__

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
merge pls

File: UnityProject/Assets/StreamingAssets/buildinfo.json [hunk 1]
```diff
@@ -1 +1 @@
-{"BuildNumber": 3921,"ForkName":"Unitystation"}
\ No newline at end of file
+{"BuildNumber": 23061519,"ForkName":"Unitystation"}
```
Comment (corp-0) [inline UnityProject/Assets/StreamingAssets/buildinfo.json]: Unintended change?

Comment (MaxIsJoe) [reply UnityProject/Assets/StreamingAssets/buildinfo.json]: Unintended

File: UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs [hunk 2]
```diff
@@ -1,8 +1,10 @@
 ﻿using System;
+using System.Diagnostics;
 using System.Linq;
 using Light2D;
 using Mirror;
 using UnityEngine;
+using Debug = UnityEngine.Debug;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs]: What are you using these namespaces for?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs]: leftovers from testing. cleared them now

**Review by corp-0 — APPROVED** [review]

**Comment by MaxIsJoe** [comment]
pr2changelog failed

**Comment by corp-0** [comment]
I removed the changelog entry. I don't think it is needed this time since players still hasn't noticed the error. 

Also we will only register fixes of errors that are present in the stable release, once I finish doing the release tag and pipeline. Will do it now.

**Pull request merged**